xarray/core/dataset.py

```diff
@@ -404,11 +420,8 @@ def copy(self, deep=False):
         Otherwise, a shallow copy is made, so each variable in the new dataset
```

Comment:
can you update this to say it makes a shallow copy of each of the component variables?